src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -70,6 +71,7 @@ function ProductDetails({ product: staleProduct }: Props) {
       <Button {...buyProps} disabled={isValidating}>
         Add to cart
       </Button>
+      <OutOfStock onSubmit={(email) => alert(email)} />
```

Comment:
[Off-topic] Food for later thought: understand how [ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) work. I feel like this component is a case for this because it only appears as the user is interacting with the SKU selector and needs (?) to be announced by assistive technologies.

Reply:
Thanks for sharing it, @filipewl. It is a live region, indeed. I guess it is something we are only going to tackle in the future (due to our priorities), but I'm to leave comments in the code about it.